pytermgui/ansi_interface.py

```diff
@@ -316,29 +333,27 @@ def save_screen() -> None:
 
     Use `restore_screen()` to get them back."""
 
-    # print("\x1b[?47h")
-    _tput(["smcup"])
+    _stdout.write("\x1b[?47h")
 
 
 def restore_screen() -> None:
     """Restore the contents of the screen saved by `save_screen()`"""
 
-    # print("\x1b[?47l")
-    _tput(["rmcup"])
+    _stdout.write("\x1b[?47l")
 
 
 def set_alt_buffer() -> None:
     """Start alternate buffer
 
     Note: This buffer is unscrollable."""
 
-    print("\x1b[?1049h")
+    _stdout.write("\x1b[?1049h")
 
 
 def unset_alt_buffer() -> None:
     """Return to main buffer from alt, restoring its original state"""
 
-    print("\x1b[?1049l")
+    _stdout.write("\x1b[?1049l")
```

Comment:
These have `print` as the function because the effects are supposed to be immediate, and you only write them to the screen one time per program execution in a normal case. Writing to `stdout` will make it buffered, which is not what these functions are meant to be.

Regardless, this is **outside of the scope**.

Reply:
I accidentally replaced commit code with code of planned commits

Reply:
I resolved it